features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
It looks like doing your suggested change with `cmsis/core_cmSecureAccess.h` allows it to build, though not sure if it actually runs. But I guess we can always start from here! I'll update this PR.